arkouda/categorical.py

```diff
@@ -180,7 +205,32 @@ def binop(self, other, op):
                                 "non-Categorical not yet implemented. " +
                                 "Consider converting operands to Categorical."))
 
-    def r_binop(self, other, op):
+    def r_binop(self, other : 'Categorical', op : str) -> pdarray:
```

Comment:
`r_binop` is not suppose to be user facing. maybe it should get underscores prepended to the name?

Reply:
Yup

Reply:
fixed!